README.md

```diff
@@ -35,11 +37,11 @@ Download
 --------
 
 ```groovy
-compile 'com.twotoasters.servos:util:1.0.0'
-compile 'com.twotoasters.servos:util-butterknife:1.0.0'
-compile 'com.twotoasters.servos:util-otto:1.0.0'
-compile 'com.twotoasters.servos:util-otto-retrofit:1.0.0'
-compile 'com.twotoasters.servos:util-picasso:1.0.0'
+compile 'com.ticketmaster.servos:util:1.0.0'
```

Comment:
Discussion:
There's been talk of using com.ticketmaster.mobilestudio as our package.  Sure we want com.ticketmaster?